src/openrct2/peep/Peep.cpp

```diff
@@ -1528,6 +1527,21 @@ bool Peep::SetName(std::string_view value)
     return false;
 }
 
+bool Peep::IsWalking() const
```

Comment:
I think this will cause confusion with the walking state. Agree that this should be renamed to add action into the name IsActionWalking same for idle to be consistent